.github/workflows/ci.yml

```diff
@@ -45,7 +45,7 @@ jobs:
 
   test_trace_decoder:
     name: Test trace_decoder
-    runs-on: ubuntu-latest
+    runs-on: zero-ci
```

Comment:
while we're at it, I noticed Jerigon job already uses `zero-ci` but takes twice longer than the second slowest job (about 15min). It may be worth considering a beefier machine, or simply reducing the range covered (b2-b8 -> b2-b5) should be enough I think.